### ISSUE TITLE
Determine features from Jetpack, Woo, Sensei, and Crowdsignal to enable in Launchpad

### ISSUE THREAD

**Issue by annezazu** [Automattic/wp-calypso#96198]
### Details

Building on the initial vision of tailored flows for WordPress.com where users were given a curated onboarding and creation experience, work is underway to bring the same approach to all goals someone might choose from. This means that any goal someone selects during onboarding will automatically result in a more tailored and specific experience with those goal(s) in mind when someone completes signing up. As a result, this work involves outlining all the valuable features we bring to our users across our various products in order to package them up in a way that streamlines onboarding and, ultimately, brings helps the user to accomplish what they want.

### Tracking p2s

p1HpG7-uTr-p2
p6q8Tx-5VL-p2
p6rkRX-7lE-p2
pabtAt-2Mw-p2

**Comment by autumnfjeld** [comment]
I think instead of a broad issue like this, I suggest want to have separate issues per Goal.    That's a clearer way to break down the work. 

For example we'd set up issues for every goal like:

"For Goal X, audit features from JP, Woo, Sensei and crowdsignal to create appropriate LP tasks"

**Comment by annezazu** [comment]
Agreed! Let's close this out as an initial sweep was done to ask for features to enable and include. At this point, going go by goal more in depth seems wise as you said.
